github/github.go

```diff
@@ -67,7 +67,7 @@ const (
 	mediaTypeTimelinePreview = "application/vnd.github.mockingbird-preview+json"
 
 	// https://developer.github.com/changes/2016-07-06-github-pages-preiew-api/
-	mediaTypePagesPreview = "application/vnd.github.mister-fantastic-preview+json"
+	mediaTypePagesPreview = "application/vnd.github.switcheroo-preview+json"
```

Comment:
I'm not sure why you are changing an existing `mediaType`, especially when the URL is old and no longer exists.

Reply:
Sorry, I was confused, now I had updated my PR